### DISCUSSION TITLE
Add setting to turn on/off volume steps and cutomize the stepping

### DISCUSSION THREAD

**Discussion by Glenta** [File-New-Project/EarTrumpet#1215] [Ideas]
When I change the volume for different applications, I like to change them in 5 volume increments. A great feature to add is to make it easier to do this by adding volume steps. Whenever dragging the volume slider, I always land on a number that is not divisible by 5, there is no real precise way to change the volume in this way except scrolling but to reach my preferred volume, I would have to drag the slider to an odd number then scroll to my preferred volume number.

By adding steps, you would be able to scroll or drag the volume to the closest step option you choose for example 5 or whatever number. When scrolling up or down by one, the volume would go up or down by 5 or whatever step chosen in settings. When dragging, the volume will go to the closest volume step, such as dragging to 32 will make the volume 30 (this is all if you choose to have a step of 5).

**Comment by sacc91** [comment]
I'd love for the scrolling step to be in a different number. For me it changes in increments of 4 but I'd rather it be 5.
